css/user.css

```diff
@@ -8,4 +8,9 @@
     width: 32px;
     height: 32px;
     padding-right: 6px;
+}
+.mimic-profile-header {
+    color: #333;

```

Comment:
This should be a theme color - you probably want `var(--text)`. Refer to modern.css for more usage. I'm also not sure how variables work across CSS files.

(I'm not doing a full review at the moment but this caught my eye.)